src/main/java/com/amazonaws/services/kinesis/leases/impl/LeaseSerializer.java

```diff
@@ -137,6 +137,18 @@ public Lease fromDynamoRecord(Map<String, AttributeValue> dynamoRecord) {
         return result;
     }
 
+    @Override
+    public Map<String, ExpectedAttributeValue> getDynamoExistantExpectation(final String leaseKey) {
+        Map<String, ExpectedAttributeValue> result = new HashMap<>();
+
+        ExpectedAttributeValue expectedAV = new ExpectedAttributeValue();
+        expectedAV.setValue(DynamoUtils.createAttributeValue(leaseKey));
+        expectedAV.setExists(true);
```

Comment:
If builder is available let's use it

https://github.com/ashwing/amazon-kinesis-client/pull/88/files#diff-4705950dfc6a2c55e5d2677225cf531fR199

Reply:
Builder is not available in this version of sdk